reference/catalog/brands_catalog.v3.yml

```diff
@@ -274,8 +282,6 @@ paths:
         **Limits**
         - 30,000 brands per store limit
       operationId: createBrand
-      parameters:
-        - $ref: '#/components/parameters/ContentType'
```

Comment:
Why was the Content-type parameter removed?

Reply:
Returned to code